pkg/components/components.go

```diff
@@ -51,6 +51,10 @@ const (
 
 	kubevirtProjectName = "KubeVirt project"
 	rbacVersionV1       = "rbac.authorization.k8s.io/v1"
+
+	// labels to apply network policies:
+	allowEgressToDNSAndAPIServerLabel  = "hco.kubevirt.io/allow-access-cluster-services"
```

Comment:
maybe the label should be renamed like:
```suggestion
	allowEgressToDNSAndAPIServerLabel  = "hco.kubevirt.io/allow-cluster-services-access"
```
so it will be with the same format as the prometheus one?